src/main/java/HomeWork_2/Array.java

```diff
@@ -0,0 +1,127 @@
+package HomeWork_2;
+
+public class Array {
+
+    int arr[];
```

Comment:
I would declare it as `private int arr[]`

Reply:
fixed